framework/src/utils/MooseEnumBase.C

```diff
@@ -157,6 +157,15 @@ MooseEnumBase::getRawNames() const
   return Moose::stringify(_items, " ");
 }
 
+std::vector<int>
+MooseEnumBase::getIDs() const
+{
+  std::vector<int> output;
```

Comment:
Note: possible optimization: you could actually reserve the right amount of space for your vector since the size of the items is known. Also, I don't see why you don't use a range based for below.